src/lib/util/command-click-action.ts

```diff
@@ -0,0 +1,75 @@
+import { setContext } from "svelte";
```

Comment:
I tracked down the animation bug – it's due to the `rill:app:ui:[key]-click-action-callbacks` being defined (`setContext`) and accessed (`getContext`) in sibling components, not in parent-child components. It was actually introduced in #405, so I'll open up a new PR to address it – to avoid conflating concerns and to keep this PR smaller.